docs/docsite/rst/playbook_guide/playbooks_reuse.rst

```diff
@@ -116,34 +116,34 @@ Each approach to re-using distributed Ansible artifacts has advantages and limit
 .. table::
    :class: documentation-table
 
-   ========================= ======================================== ========================================
-   ..                        Include_*                                Import_*
-   ========================= ======================================== ========================================
-   Type of reuse             Dynamic                                  Static
+   ========================= ================================================== ========================================
+   ..                        Include_*                                          Import_*
+   ========================= ================================================== ========================================
+   Type of reuse             Dynamic                                            Static
 
-   When processed            At runtime, when encountered             Pre-processed during playbook parsing
+   When processed            At runtime, when encountered                       Pre-processed during playbook parsing
 
-   Task or play              All includes are tasks                   ``import_playbook`` cannot be a task
+   Task or play              All includes are tasks                             ``import_playbook`` cannot be a task
 
-   Task options              Apply only to include task itself        Apply to all child tasks in import
+   Task options              Apply only to include task itself                  Apply to all child tasks in import
 
-   Calling from loops        Executed once for each loop item         Cannot be used in a loop
+   Calling from loops        Executed once for each loop item                   Cannot be used in a loop
 
-   Using ``--list-tags``     Tags within includes not listed          All tags appear with ``--list-tags``
+   Using ``--list-tags``     Tags within includes not listed                    All tags appear with ``--list-tags``
 
-   Using ``--list-tasks``    Tasks within includes not listed         All tasks appear with ``--list-tasks``
+   Using ``--list-tasks``    Tasks within includes not listed                   All tasks appear with ``--list-tasks``
 
-   Notifying handlers        Cannot trigger handlers within includes  Can trigger individual imported handlers
```

Comment:
@kpinc The original phrasing seems correct to me. Can you provide examples of what behavior are you referring to?

Reply:
I see. Do you think `Cannot notify handlers defined within includes` or to be consistent with the statement about imports: `Cannot notify individual included handlers` be better?